lib/stepmod/utils/converters/term.rb

```diff
@@ -1,22 +1,22 @@
 # frozen_string_literal: true
 
-require 'stepmod/utils/converters/synonym'
+require "stepmod/utils/converters/synonym"
 
 module Stepmod
   module Utils
     module Converters
       class Term < ReverseAdoc::Converters::Base
         def convert(node, state = {})
           first_child = node.children.find do |child|
-                          child.name == 'text' && !child.text.to_s.strip.empty?
-                        end
+            child.name == "text" && !child.text.to_s.strip.empty?
+          end
           unless first_child &&
-                  node.text.split(';').length == 2 &&
-                  defined?(Stepmod::Utils::Converters::Synonym)
+              node.text.split(";").length == 2 &&
+              defined?(Stepmod::Utils::Converters::Synonym)
             return "=== #{treat_children(node, state).strip}"
           end
 
-          term_def, alt = node.text.split(';')
+          term_def, alt = node.text.split(";")
           alt_xml = Nokogiri::XML::Text.new(alt, Nokogiri::XML::Document.new)
           converted_alt = Stepmod::Utils::Converters::Synonym.new.convert(alt_xml)
           "=== #{term_def}\n\n#{converted_alt}"
```

Comment:
This should go into Glossarist as well.

Reply:
@ronaldtse this is a stepmod converter, and as such it does not make sense to move it to Glossarist. What exactly from it should go into Glossarist?

Reply:
`"=== #{term_def}\n\n#{converted_alt}"` is writing out Metanorma AsciiDoc. We should do: XML => Glossarist => Metanorma AsciiDoc.

Reply:
Discussed with @ronaldtse, decided to left it like this